minitest/use_cases/company.py

```
@@ -0,0 +1,41 @@
+        id=uuid.uuid4(), 
+        user_id=user, 
+        company_name=company_name, 
+        company_address=company_address,
+        total_staff=total_staff,
+        created_at=datetime.now(),
+        updated_at=None
+        )
+    company = company_repo.create_company(company, user)
+    return company
+
+def change_company(uuid, company_name, company_address, total_staff):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)